modelling/src/test/java/org/axonframework/modelling/entity/PolymorphicEntityModelTest.java

```diff
@@ -55,46 +55,50 @@ class PolymorphicEntityModelTest {
     public static final QualifiedName CONCRETE_ONE_EVENT = new QualifiedName("ConcreteOneEvent");
     public static final QualifiedName CONCRETE_TWO_EVENT = new QualifiedName("ConcreteTwoEvent");
 
-    private final EntityModel<ConcreteTestEntityOne> concreteTestEntityOneEntityModel = Mockito.mock(EntityModel.class);
-    private final EntityModel<ConcreteTestEntityTwo> concreteTestEntityTwoEntityModel = Mockito.mock(EntityModel.class);
+    private final EntityMessagingMetamodel<ConcreteTestEntityOne> concreteTestEntityOneEntityMetamodel = Mockito.mock(
+            EntityMessagingMetamodel.class);
+    private final EntityMessagingMetamodel<ConcreteTestEntityTwo> concreteTestEntityTwoEntityMetamodel = Mockito.mock(
+            EntityMessagingMetamodel.class);
     private final EntityCommandHandler<AbstractTestEntity> entityInstanceCommandHandler = Mockito.mock(
             EntityCommandHandler.class);
     private final CommandHandler entityCreationalCommandHandler = Mockito.mock(CommandHandler.class);
     private final EntityEvolver<AbstractTestEntity> entityEvolver = Mockito.mock(EntityEvolver.class);
 
-    private EntityModel<AbstractTestEntity> polymorphicModel;
+    private EntityMessagingMetamodel<AbstractTestEntity> polymorphicModel;
```

Comment:
Nit: Pretty sure this should be `polymorphicMetamodel` now.

Reply:
Correct, renamed!